asciidoc-java/src/main/java/io/yupiik/asciidoc/renderer/html/AsciidoctorLikeHtmlRenderer.java

```diff
@@ -880,15 +880,18 @@ protected void visitToc(final Body body) {
 
     // todo: enhance
     protected void visitXref(final Macro element) {
+        final var outFileSuffix = configuration.getAttributes().getOrDefault("outfilesuffix", ".html");
```

Comment:
Shouldnt also page attributs be read (header if not null) cause config can be static for all pages

Reply:
TBH I don't know, I hesitated between both..

Reply:
Any reason why for `:imagesdir:` we used the conf?

Reply:
tempted to say anywhere it should be a header or conf and was a legacy from the first original impl

Reply:
ok I let you figure it out in a follow up PR :)